saylua/modules/explore/views.py

```diff
@@ -13,11 +13,6 @@ def home():
 
 
 @login_required
-def battle():
-    return render_template("battle.html", bg_num=randint(1, 21))
-
-
-@login_required
```

Comment:
It was was a deletion of login_Required on battle(), but git interprets the blocking weird.